ProcessLib/SmallDeformationWithLIE/Common/FractureProperty.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
```

Comment:
I would use the include guard in this PR. Changes to pragma once should be done in another PR.